docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -0,0 +1,113 @@
+Run LLMs with OpenVINO GenAI Flavor on NPU
+==========================================
+
+.. meta::
+   :description: Learn how to use the OpenVINO GenAI flavor to execute LLM models on NPU.
+
+This guide provide some extra details on how to utilize NPU with GenAI
+flavor. Please refer to
+:doc:`install OpenVINO GenAI <../../get-started/install-openvino/install-openvino-genai>`
+for more information on OpenVINO GenAI.
+
+Export an LLM model via Hugging Face Optimum-Intel
+##################################################
+
+1. Install the correct components for exporting model:
+   .. code-block:: text
+   
+   # requirements.txt
```

Comment:
Formatting is broken here at least when you view it on Github. I think a newline is required before this `.. code-block` thing.

Reply:
Fixed